.github/workflows/slsa-goreleaser.yml

```diff
@@ -0,0 +1,35 @@
+name: SLSA go releaser
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "*"
+
+permissions: read-all
+
+jobs:
+  # Generate ldflags dynamically.
+  args:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.ldflags.outputs.version }}
+    steps:
+      - id: checkout
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v3.0.2
```

Comment:
These comments fall out of date pretty quickly when dependabot bumps them. Can you remove that part?

Reply:
Fixed. (Note: Renovatebot PRs update the comment, but not dependabots's.)